src/components/LogIn/LogIn.test.tsx

```diff
@@ -53,7 +53,7 @@ describe("LogIn", () => {
   it("renders an IdentityProvider login UI if the user is not logged in", () => {
     const state = rootStateFactory.build({
       general: generalStateFactory.withConfig().build({
-        visitURL: "I am a url",
+        visitURLs: ["I am a url"],
```

Comment:
I've made this an array as we can have more than one login request at a time now.